dimagi/utils/logging.py

```diff
@@ -29,8 +29,11 @@ def notify_error(message):
 
 
 def notify_exception(request, message=None, details=None):
+    if request is not None:
+        message = message or request.path
     notify_logger.error(
-        'Notify Exception: %s' % (message or request.path),
+        'Notify Exception: %s' % (message
+                                  or "No message provided, fix error handler"),
```

Comment:
This looks like a good thing to have in case both message and request are None, but I don't think this will fix the issue referenced in the ticket, which is that the notify_logger is not using this string as the subject of the email that gets sent like it used to.